packages/web/src/components/range/DynamicRangeSlider.d.ts

```diff
@@ -17,6 +17,11 @@ export interface DynamicRangeSliderProps extends CommonProps {
 	rangeLabels?: (...args: any[]) => any;
 	react?: types.react;
 	showHistogram?: boolean;
+	snapPoints?: number[] | (start: number, end: number, stepValue: number) => number[];
```

Comment:
I am wondering if we should just stick to one type here - it can always be a function which can return the static list of numbers if you wish to (irrespective of the given params). This way, it will be easier for anybody using this because now they always have to stick to one format and it will be easily maintainable from the development/docs standpoint. Thoughts?

Reply:
Agreed. This makes more sense. Probably unlikely that anyone would use an array literal here anyway.